### DISCUSSION TITLE
[Invites] Triage Team

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/open-sauced#638] [General] [answered]
The Triage team is inspired by [expressjs/express](https://github.com/expressjs/express/blob/master/Triager-Guide.md). This is a place for those who would like to contribute to the project but might not know where to start. 

If you would like to assist in identifying bugs, reviewing pull requests, or just looking to be a part of a community. Please let us know with a reply and I can add you to the team. 

**The requirement for being a member of the team is having an [opensauced.pizza](https://opensauced.pizza/) account.** 

Please review the [Triage Guide](https://github.com/open-sauced/.github/blob/main/TRIAGE_GUIDE.md) for details and expectations.

[Bug Board](https://github.com/open-sauced/open-sauced/projects/1)

**Comment by sayanisur** [comment]
I would be interested for the same

**Comment by sujitagarwal** [comment]
I'm interested!

**Reply by bdougie** [reply]
Awesome, invite sent. Be sure to join us in the [Discord](https://discord.gg/gZMKK5q) too.

**Reply by akshay5995** [reply]
I'm interested too✋🏿

**Reply by bdougie** [reply]
Hey @sujitagarwal, you are the only triage team member in the [Discord](https://discord.gg/gZMKK5q). Feel free to come join us there 😄

**Reply by sujitagarwal** [reply]
Oops. Joining now.

**Comment by GianCastle** [comment]
I'm interested

**Reply by bdougie** [reply]
Invite sent. Be sure to join us in the [Discord](https://discord.gg/gZMKK5q) too.

**Reply by GianCastle** [reply]
Thanks, mates!!

**Comment by ColomboJuan** [comment]
I'm interested!

**Reply by bdougie** [reply]
Invite sent

**Comment by stubrew24** [comment]
Hello, I'm interested

**Reply by bdougie** [reply]
Invite sent

**Comment by JapneetSingh5** [comment]
Hi, I’m interested!

**Reply by bdougie** [reply]
invited. Be sure to join us in the [Discord](https://discord.gg/gZMKK5q) to say hello.

**Comment by kunal1244** [comment]
Hey! I'm interested.

**Reply by bdougie** [reply]
Hey you are in! Be sure to join the Discord too!

**Comment by bdougie** [comment, marked as answer]
At this time we will be pausing on adding members to the Triage team through this Discussion. 

If you have an interest, please let us know in the [Discord](https://discord.gg/gZMKK5q). Thanks!
